.github/workflows/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+### Purpose
+Describe the problem or feature in addition to a link to the issues.
+
+### All Submissions:
+* [ ] I have read the CONTRIBUTING document.
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
+* [ ] Does your filename follow the naming Conventions?
+* [ ] Have you linked your PR to an Issue?
```

Comment:
Remove this because there is a heading under it asking to link a PR.

Reply:
done